src/main/java/Duke.java

```diff
@@ -1,10 +1,185 @@
+
 public class Duke {
+    // types of command
+    enum COMMANDS {
+        COMMAND_INIT,
+        COMMAND_ADD,
+        COMMAND_LIST,
+        COMMAND_MARKDONE,
+        COMMAND_EXIT,
+        COMMAND_UNRECOGNIZE,
+    }
+
+    // Formatting
+    private final static String formatDashes = "----------------------------------------------------------------------";
+    private final static String formatTwoTabs = "    ";
+    private final static String formatFourTabs = "        ";
+
+    // String object to hold any user input or output message
+    private static String message = "";
+    private static String[] splitMessage;
+    private static String command = "";
+
+    private static Scanner in = new Scanner(System.in);
+
+    // Characters for tick and cross
+    public static char tick = '\u2713';
+    public static char cross = '\u2718';
+
+    // Boolean for exit app
+    private static boolean isExit = false;
+
+    // User task list
+    private static List<Task> taskList = new ArrayList<>();
+
+    // Print message with formatting
+    private static void reply(String msg) {
+        System.out.println(msg + formatDashes);
+    }
+
+    private static void printWelcomeMessage() {
+        // Welcome message
+        message = formatTwoTabs + "Hello! I'm Jay. Today is " + DateTimeManager.getDate() + ", " + DateTimeManager.getDay() +
+                ". The " + "time now is " + DateTimeManager.getTime() + "." + System.lineSeparator() + formatTwoTabs + "What can I" +
+                " do for you?" + System.lineSeparator();
+        reply(message);
+    }
+
+    private static void printExitMessage() {
+        message = formatTwoTabs + "Bye! Hope to see you again soon!" + System.lineSeparator();
+        reply(message);
+    }
+
+    // Add task
+    private static void AddTask() {
+        Task newTask;
+
+        message = formatTwoTabs + "Added ";
+        // Create new task object based on the command type
+        switch(command.toUpperCase()) {
+        case "DEADLINE":
+            splitMessage = splitMessage[1].split("/by ", 2);
+            newTask = new Deadline(splitMessage[0], splitMessage[1]);
+            message += "a deadline task ";
+            break;
+        case "EVENT":
+            splitMessage = splitMessage[1].split("/at ", 2);
+            newTask = new Event(splitMessage[0], splitMessage[1]);
+            message += "an event ";
+            break;
+        case "TODO":
+        default:
+            newTask = new Todo(splitMessage[1]);
+            message += "a todo task ";
+            break;
+        }
+
+        // add to the list
+        taskList.add(newTask);
+        // print out the newly added task
+        message += newTask + "!" + System.lineSeparator() + formatTwoTabs + "Now you have " + taskList.size() +
+                " task(s) in the list" + System.lineSeparator();
+        reply(message);
+    }
+
+    // List task
+    private static void ListTask() {
+        int index = 1;
```

Comment:
I see the index was declared here, but only used in the else statement. Would it be better if we declared it over there instead?

Reply:
Noted this too, thanks!